signup-ui/src/utils/mediaDevicePermission.ts

```diff
@@ -0,0 +1,8 @@
+export const requestCameraPermission = async () => {
+  try {
+    await navigator.mediaDevices.getUserMedia({ video: true });
+    console.log("Camera permission granted");
```

Comment:
do we need console.log?

Reply:
It is to check if the permission is denied or granted for debugging purpose.